src/python/cli_new/lib/cli/http.py

```diff
@@ -19,70 +19,63 @@
 """
 
 import json
-import urllib.request
-import urllib.error
-import urllib.parse
-import time
+from urllib.parse import urlencode
+import urllib3
 
 import cli
 
 from cli.exceptions import CLIException
 
+urllib3.disable_warnings()
```

Comment:
Why is this necessary?

Reply:
I'm getting a lot of non pretty warning if I'm using a self signed certificate. But because of the possibility to set "ssl_verify", didn't think it made sense to issue the warning as well.

Reply:
Then maybe add a comment explaining it?

Reply:
Will do it :-)